src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -942,6 +942,16 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ProjectCapability Remove="ReferenceManagerAssemblies" />
   </ItemGroup>
 
+  <!--
+  ============================================================
+                            Set ImportWindowsDesktopTargets
+  ============================================================
+  -->
+
+  <PropertyGroup Condition=" '$(ImportWindowsDesktopTargets)' == '' and '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '5.0')) and '$(TargetPlatformIdentifier)' == 'Windows' and ('$(UseWpf)' == 'true' Or '$(UseWindowsForms)' == 'true')">
```

Comment:
@dsplaisted could you check this change again? I added check on TargetPlatformIdentifier, and _TargetFrameworkVersionWithoutV >= 5.0 on this condition

WPF side just merged.

Reply:
So the new syntax only applies to net5.0-windows. If you use Microsoft.NET.Sdk.WindowsDesktop, that is still the same

Reply:
Since this is only importing the WindowsDesktop targets if `UseWPF` or `UseWindowsForms` are true, the default `PlatformVersion` logic that [sarah added](https://github.com/dotnet/wpf/pull/3177/files) won't apply to projects targeting Windows but not using WPF or Windows Forms.

I think what we should do is just move that logic into the dotnet/sdk repo.  Can that be done as part of this PR? (We'd delete the logic from the wpf repo later).

Nit: Can you wrap the condition onto multiple lines for readability?

Reply:
Done wrapping and added default PlatformVersion logic